andela/healthchecks_asgards
# #135537529 Add sendgrid to requirements.txt

#### What does this PR do?
Fixes the import error that causes healthchecks heroku app to crash

#### Description of Task to be completed?
Add sendgrid package to requirements.txt
Add an environment variable to hold the sendgrid token

#### How should this be manually tested?
Go to [healthchecks_asgards](http://healthchecks-aasgards.herokuapp.com/) on heroku.
Enter your email in the `Enter email to receive alerts` field and click enter
Check if an email was sent to your inbox


#### What are the relevant pivotal tracker stories?
#135537529
